warehouse/models/mart/ad_hoc/fct_scheduled_service_by_daypart.sql

```diff
@@ -0,0 +1,107 @@
+{{ config(materialized='table') }}
+
+WITH dim_shapes_arrays AS (
```

Comment:
Also sorry I think you can actually remove these CTEs at the top for shapes and routes since they are no longer used?